internal/driver/driver.go

```diff
@@ -122,6 +140,95 @@ func (d *Driver) createClientOptions() ([]opcua.Option, error) {
 	return opts, nil
 }
 
+// Gets the username and password credentials from the configuration.
+func (d *Driver) getCredentials(secretPath string) (config.Credentials, error) {
```

Comment:
same as above

Reply:
you mean rename d? My answer would be same as above :D